src/Kernel32/Kernel32.cs

```diff
@@ -439,6 +439,64 @@ public static partial class Kernel32
+        /// <para>
+        /// You can specify a handle to one of the following types of objects: access token, console input buffer, console screen buffer, event, file, file mapping, job, mailslot, mutex, pipe, printer, process, registry key, semaphore, serial communication device, socket, thread, or waitable timer.
+        /// </para>
+        /// </param>
+        /// <param name="dwMask">
+        /// A mask that specifies the bit flags to be changed. Use the same constants shown in the description of <paramref name="dwFlags"/>.
+        /// </param>
+        /// <param name="dwFlags">
+        /// Set of bit flags that specifies properties of the object handle. This parameter can be 0 or one or more of the values of <see cref="HandleFlags"/>.
+        /// </param>
+        /// <returns>
+        /// <para>
+        /// If the function succeeds, the return value is nonzero.
+        /// </para>
+        /// <para>
+        /// If the function fails, the return value is zero. To get extended error information, call <see cref="GetLastError"/>.
+        /// </para>
+        /// </returns>
+        /// <remarks>
+        /// To set or clear the associated bit flag in <paramref name="dwFlags"/>, you must set a change mask bit flag in <paramref name="dwMask"/>.
+        /// </remarks>
+        [DllImport(api_ms_win_core_handle_l1_1_0, SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool SetHandleInformation([In] SafeHandle hObject, HandleFlags dwMask, HandleFlags dwFlags);
+
+        /// <summary>
+        /// Retrieves certain properties of an object handle.
+        /// </summary>
+        /// <param name="hObject">
+        /// <para>
+        /// A handle to an object whose information is to be retrieved.
+        /// </para>
+        /// <para>
+        /// You can specify a handle to one of the following types of objects: access token, console input buffer, console screen buffer, event, file, file mapping, job, mailslot, mutex, pipe, printer, process, registry key, semaphore, serial communication device, socket, thread, or waitable timer.
+        /// </para>
+        /// </param>
+        /// <param name="lpdwFlags">
+        /// A pointer to a variable that receives a set of bit flags that specify properties of the object handle or 0. The values are defined at <see cref="HandleFlags"/>.
+        /// </param>
+        /// <returns>
+        /// <para>
+        /// If the function succeeds, the return value is nonzero.
+        /// </para>
+        /// <para>
+        /// If the function fails, the return value is zero. To get extended error information, call <see cref="GetLastError"/>.
+        /// </para>
+        /// </returns>
+        [DllImport(api_ms_win_core_handle_l1_1_0, SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern unsafe bool GetHandleInformation([In] SafeHandle hObject, [Out] out HandleFlags* lpdwFlags);
```

Comment:
Please remove `[In]` and `[Out]` here as well. The `out` modifier as well should be removed since this isn't a pointer to a pointer. It's just a pointer that we must pass *in* so that Win32 knows where to write the `DWORD`. I'm pretty sure this would fail as written here.

Also please add `[Friendly(FriendlyFlags.Out)]` to the `HandleFlags*` parameter so that an overload is generated that has an `out HandleFlags` parameter.